### PR TITLE
Add IntelliJ code style configs

### PR DESCRIPTION
**Pull request by gegy1000**
Bikeshedding moment. Feel free to give opinions if you hate these formatting options :P

### CODE REVIEW

**Comment by hYdos** [comment]
use netbeans

**Comment by Restioson** [comment]
add config for rustfmt

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+  </state>
+</component>
```
Comment (ramidzkh) [inline .idea/codeStyles/codeStyleConfig.xml:5]: newline at eof

File: .idea/inspectionProfiles/Project_Default.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="CodeBlock2Expr" enabled="true" level="WEAK WARNING" enabled_by_default="true" />
+    <inspection_tool class="MissingOverrideAnnotation" enabled="true" level="WARNING" enabled_by_default="true">
+      <option name="ignoreObjectMethods" value="true" />
+      <option name="ignoreAnonymousClassMethods" value="false" />
+    </inspection_tool>
+    <inspection_tool class="UnqualifiedFieldAccess" enabled="true" level="WARNING" enabled_by_default="true" />
+    <inspection_tool class="UnqualifiedMethodAccess" enabled="true" level="WARNING" enabled_by_default="true" />
+  </profile>
+</component>
```
Comment (ramidzkh) [inline .idea/inspectionProfiles/Project_Default.xml:12]: newline at eof

**Pull request merged**